probe/probe.go

```diff
@@ -156,8 +156,17 @@ func (p *Probe) StartProbing() error {
 		}
 	}
 
-	tickerProbe := time.NewTicker(time.Duration(millisecondInMinute/p.probeRatePerMin) * time.Millisecond)
-	tickerDurabilityProbe := time.NewTicker(time.Duration(millisecondInMinute/p.durabilityProbeRatePerMin) * time.Millisecond)
+	var tickerProbe, tickerDurabilityProbe *time.Ticker
```

Comment:
Can't we have a boolean that activate or not the check instead of a fake tikcer at 9999999 hours

Reply:
:'( it was so beautiful